features/onbekend_waardes.feature

```diff
@@ -501,17 +603,14 @@ Rule: vertalen (onbekend)waarden naar indicator
   - elke waarde voor overlijdensdatum (incl. 00000000) geeft indicatieOverleden met de waarde true
 
   Abstract Scenario: indicatieVestigingVanuitBuitenland en vanuitVerblijfplaatsOnbekend bij datum vestiging '<datum>' uit land '<land>'
```

Comment:
Is dit geen vreemde scenario naam als 'vanuitVerblijfplaatsOnbekend' in 4 van de 6 voorbeelden geen rol speelt?
Dat veld is ook geen illustratie van de Rule in regel 599 dus hoort het hier wel thuis?

Reply:
Goede opmerking. Ik heb deze ondergebracht in de map immigratie in 2 aparte scenario's en hier weggehaald.